### DISCUSSION TITLE
Will twitter be a long lasting company?

### DISCUSSION THREAD

**Discussion by Goodlooking818** [community/community#39825] [New to GitHub]
Will twitter be a long lasting company or just fade out?

**Comment by ossanna16** [comment]
Hi @Goodlooking818,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Twitter. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
